src/openrct2-ui/windows/LoadSave.cpp

```diff
@@ -237,20 +238,22 @@ static const char* getFilterPatternByType(const int32_t type, const bool isSave)
     return "";
 }
 
+//Checks for path existance
+bool IsPathExist(const std::string &s)
+{
+    struct stat buffer;
+    return (stat (s.c_str(), &buffer) == 0);
```

Comment:
`stat` is not available for every platform. You instead use `platform_directory_exists` which is already implemented correctly for each platform.